elastic/integrations
# Add state_job data_stream in kubernetes package

## What does this PR do?

This PR enhances Kubernetes package with state_job data_stream after merging of [#26479](https://github.com/elastic/beats/pull/26479)

## Checklist



## How to test this PR locally

1. After bringing up a kind k8s cluster and an elastic stack run 
`ep test system --data-streams state_job -v`  inside `integrations/packages/kubernetes/data_stream` dir
2.  From Kibana UI create a policy and add `Kube-state-metrics` integration. Turn on state_job metricset and apply the policy to a running fleet managed agent. [(howto)](https://github.com/elastic/beats/tree/master/deploy/kubernetes/elastic-agent-managed) Check in `discover` panel the metrics coming from that metricset.
